src/char/char.h

```diff
@@ -29,6 +29,19 @@ enum {
 	TABLE_GUILD_STORAGE,
 };
 
+typedef enum e_makechar_error {
+	ERROR_MK_SQL = -255,
+	ERROR_INVALID_SLOT,
+	ERROR_MAXLIMIT,
+	ERROR_NAME_RESERVED,
+	ERROR_NAME_LEN,
+	ERROR_NAME_INVALID_CHAR,
+	ERROR_NAME_DUPLICATE,
+	ERROR_CREATION_DISABLE,
+
+	ERROR_NO_ERROR = 0,
+} e_makechar_error;
```

Comment:
any better prefix than `ERROR`? just because `ERROR` is too common. replace it with `MAKECHAR` maybe?